taxonomy-editor-frontend/src/constants.js

```diff
@@ -17,3 +17,189 @@ function taxonomyApiUrlFromUi(location) {
 }
 export const API_URL = taxonomyApiUrlFromUi(window.location);
 export const ENTER_KEYCODE = 13;
+export const ISO_CODES ={
```

Comment:
could we get this from the library? I think it is going to be difficult to maintain this